packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/BuyQuote.tsx

```diff
@@ -252,15 +247,6 @@ export const BuyQuote = ({ className, quote, wantCrypto }: QuoteProps) => {
                         />
                     </Value>
                 </Column>
-                <Column>
-                    <Heading>
-                        <Translation id="TR_BUY_FEES" />{' '}
```

Comment:
This translation string can also be removed from [messages.ts](https://github.com/trezor/trezor-suite/pull/11801/files#diff-988baecac61429698981f7b86b979d1627f33f484918f65402c4b345b5c57cba)

Reply:
This one will be probably used again in next iteration

Reply:
But right now we won't use us it and nightly checks for unused translations will fail

Reply:
ok, removed https://github.com/trezor/trezor-suite/pull/11801/commits/74ee4077cec2dcc0a40478479d33226839f16ba3

Reply:
Thanks, please squash it and we can merge it